frodo.js

```diff
@@ -1,126 +1,130 @@
-/**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.